tsconfig.packages.json

```diff
@@ -5,6 +5,11 @@
     "esModuleInterop": true,
     "sourceMap": true,
     "composite": true,
-    "strict": true
+    "strict": true,
+
+    "paths": {
+      "@botpress/messaging-base": ["./packages/base/src/index.ts"],
+      "@botpress/messaging-engine": ["./packages/engine/src/index.ts"]
+    }
```

Comment:
I think we can just put this info in the jest.config file, and not have this affect our experience when we write code since vscode uses the tsconfig to decide imports etc.